flytekit/core/base_task.py

```diff
@@ -462,6 +463,7 @@ def __init__(
         environment: Optional[Dict[str, str]] = None,
         disable_deck: Optional[bool] = None,
         enable_deck: Optional[bool] = None,
+        decks: Optional[Tuple[str, ...]] = None,
```

Comment:
This is my preferred default, but I'm open to other opinions:

```suggestion
        decks: Optional[Tuple[str, ...]] = ("source_code", "dependencies"),
```

Reply:
For me, it's more about consistency. If I wanted to debug `PythonTask`, then I'll have to go back to `def task` to figure out that the default is actually ("Source Code", "Dependencies").